### ISSUE TITLE
Dark Mode Email- Implement and Test

### ISSUE THREAD

**Issue by alisonuxdesign** [ezl/dailyidea.com#557]
The goal of this ticket is to render our emails in a more attractive form in both Light mode and Dark mode in an email client. Below we have three instances: 
![Screen Shot 2020-09-20 at 3 31 03 PM](https://user-images.githubusercontent.com/63747360/93722756-4d3faf80-fb56-11ea-9a2f-96433b6c2f83.png)

A. This instance is what the user would see in a light mode aka "Regular" email client. 

B. In this instance you can see that we have "dark-mode" proofed our emails. This is the exact same email with the exact same elements as the light mode email but with al colors inverted. 

C. In this email you can see that we are dictating what Dark mode renders. We have different colors in the image.  

I have attached all images of this email below: 
[Emails Dark mode.zip](https://github.com/ezl/dailyidea.com/files/5252733/Emails.Dark.mode.zip)

To implement please: 
1. Create email
2. send to yourself and to alisonuxdesign@gmail.com to test!

**Comment by vedmant** [comment]
@ezl @alisonuxdesign I think we had already similar card about this, I remember I explained that there is no proper way to make dark mode emails. Email clients like gmail just change email styles themselves to invert colors and make text visible on background. There are just few clients that actually support dark media query and thay are not that popular: Apple Mail (MacOS), iPhone / iPad (iOS 13) and Hey.com . This is all clients that support media queries. Not sure if we need to apply efforts just for that clients.

**Comment by ezl** [comment]
@vedmant correct - i know we previously had a ticket for this.

however, since then, @alisonuxdesign and I did more research on dark mode, and what we've established is:

(A) like you said, email clients or operating systems can do whatever they want

(B) however, there ARE dark mode tags that are supported by SOME clients, even if not all

(C) one way to handle this is to create 2 versions:

1. a "light mode" email that renders exactly how we want in a light mode, but that will render "acceptably" even if the email client or OS inverts colors using CSS. This is done by using outlined images mostly, and being more selective about the colors that are used in the "light mode" version.

The largest segment of our users are gmail users most likely, which does NOT support the media query, and implements a full invert. For this segment, the media query won't work, but the new assets and "light mode" design are intended to cascade gracefully to an acceptable full inversion by the email client.

2. ALSO ship a "dark mode" email and styles and images, that will be rendered in the event that the email client respects the `prefers-color-scheme: dark` media query. Even though this is only supported by a few browsers, Apple OS and email clients respect it. 

Our google analytics shows that ~35% of our users are on an apple OS or browser. This is consistent with the [Litmus email market share report](https://www.litmus.com/blog/infographic-the-2019-email-client-market-share/).

So that means we'll still be providing a better experience to a meaningful percentage of users by providing a deliberate dark mode design here via the media query.


--

TLDR: These emails are better, given our new understanding of which clients support dark mode and which ones don't.

However, if there is a fundamental reason why it is difficult to implement these in MJML, or if it SIGNIFICANTLY increases complexity in a way that you feel this may be a bad technical choice, I think we should still discuss.

Should we not do this @vedmant ?

**Comment by vedmant** [comment]
@ezl What I can do here is to add media queries that will show different logo, main image and invert colors for clients that support it. It's not an issue with MJML, it supports media queries.
It also might be difficult to test, I'll need to setup clients, also I can only test on MacOs, I don't have iOs device.

**Comment by vedmant** [comment]
- [ ] require_login_template.mjml
- [ ] magic_link_template_with_comment.mjml
- [ ] magic_link_template_with_idea_save.mjml
- [ ] require_login_template_mobile.mjml
- [ ] require_login_template_with_comment.mjml
- [ ] require_login_template_with_idea_save.mjml
- [ ] signup_template.mjml

**Comment by vedmant** [comment]
@ezl I updated login email, I tested it on macos and hey.com, it works ok, but I don't have iOs to test it on, could you check if it works ok on iOs. Then I'll make edits to all other emails.

**Comment by ezl** [comment]
@vedmant I'll test it on iOS, can you send me the email to my gmail?

Also, for now, for this ticket, you  just need to implement for the 1 email that is included in the task (the login email) -- not for all emails.

After that, @alisonuxdesign and I will review and assess whether we should roll it out for all emails.

**Comment by vedmant** [comment]
@ezl I see, ok, then I'm creating a PR
